<?xml version="1.0" encoding="UTF-8"?><record_update table="syntax_editor_macro">
    <syntax_editor_macro action="INSERT_OR_UPDATE">
        <comments>Client-side try/catch</comments>
        <name>try</name>
        <sys_class_name>syntax_editor_macro</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2021-10-05 05:28:49</sys_created_on>
        <sys_id>88d204492fc33010c3fe54492799b6da</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>try</sys_name>
        <sys_package display_value="NeedIt" source="x_58872_needit">6ead8e780f603200cd674f8ce1050ed1</sys_package>
        <sys_policy/>
        <sys_scope display_value="NeedIt">6ead8e780f603200cd674f8ce1050ed1</sys_scope>
        <sys_update_name>syntax_editor_macro_88d204492fc33010c3fe54492799b6da</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2021-10-05 05:28:49</sys_updated_on>
        <text>try {&#13;
&#13;
}&#13;
&#13;
catch(err) {&#13;
    g_form.addErrorMessage("A runtime error occurred: " + err);&#13;
}</text>
    </syntax_editor_macro>
</record_update>
